frame/staking/src/tests.rs

```diff
@@ -5137,9 +5143,11 @@ fn change_of_max_nominations() {
 				vec![(70, 1), (60, 1)]
 			);
 
+			// TODO: This part has to be removed, we need to be able to decode old nominations.
```

Comment:
Same here, but this is easier. We just remove a condition from chill_other and no longer allow to chill un-decodables.

Reply:
I think a good idea here is to introduce an error when trying to act upon un-decodable nominations as we can't recover from this. I'll do that for both re-nominations and chill_other.